src/components/Subthoughts.tsx

```diff
@@ -105,15 +105,17 @@ const mapStateToProps = (state: State, props: SubthoughtsProps) => {
     showHiddenThoughts,
   } = state
 
+  const resolvedPath = props.path ?? unroot(props.simplePath)
```

Comment:
What requires that `props.simplePath` be unrooted? You should probably move this downstream, if it's needed at all. We should only unroot paths during concatenation, otherwise we could end up with an invalid empty `Path`.

Reply:
Got it.